task.sh

```diff
@@ -6,38 +6,61 @@
 #   dir2
 #   dir3
 #       dir4
-
+mkdir task
```

Comment:
Можно так одной командой сделать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```